arangod/Graph/Graph.cpp

```diff
@@ -146,6 +160,20 @@ Graph::Graph(TRI_vocbase_t& vocbase, std::string&& graphName,
   TRI_ASSERT(_rev.empty());
 
   if (info.hasKey(StaticStrings::GraphEdgeDefinitions)) {
+    // TODO: Check why we're landing here and not in SmartGraphEE - Cleanup!
+    LOG_DEVEL << "do not forget me - 2";
+    if (options.isObject()) {
```

Comment:
This also looks like it is in the Wrong position.
The satellites are independent of GraphEdgeDefinitions